CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ This version adds a dependency on Swift.
     - Rename `SentryOptions.enablePreWarmedAppStartTracking` to `enablePreWarmedAppStartTracing`
     - Rename `SentryOptions.enableFileIOTracking` to `enableFileIOTracing`
     - Rename `SentryOptions.enableCoreDataTracking` to `enableCoreDataTracing`
+- SentrySDK.close calls flush (#2453)
```

Comment:
Is this really a breaking change? Or should this just be moved to the fixes section?

Reply:
It's now a blocking call. Previously, it wasn't. Would you consider that as a breaking change or not, @kevinrenskers?

Reply:
I don't think so, plus it doesn't actually say that, in case you do think that makes it a breaking change. I think we'd better move it or explain how it's breaking?

Your call though.

Reply:
I added that it is blocking. Imagine you call this somewhere on the main thread and have plenty of envelopes in the queue. Previously, this call was fast, and now it blocks your main thread for 1 second. I rather point it out as a breaking change.

Reply:
That makes perfect sense!